Oracle/backup_cdm.py

```diff
@@ -0,0 +1,91 @@
+''' backup_cdm - back up cdm tables that have at least one row
+'''
+# Allow --dry run even without cx_Oracle
+try:
+    from cx_Oracle import DatabaseError
+except:
+    pass
+
+CDM3_TABLES = ['HARVEST', 'DEMOGRAPHIC', 'ENCOUNTER', 'DIAGNOSIS',
+               'CONDITION', 'PROCEDURES', 'VITAL', 'ENROLLMENT',
+               'LAB_RESULT_CM', 'PRESCRIBING', 'DISPENSING']
+
+
+def main(get_cursor):
+    cursor = get_cursor()
+
+    for table in CDM3_TABLES:
+        if has_rows(cursor, table):
+            drop_triggers(cursor, table)
+            drop(cursor, table + '_BAK')
+            rename(cursor, table, table + '_BAK')
+
```

Comment:
in retrospect, the relevant capability is clearly something with `has_rows`, `drop`, and `rename`:

```
db = get_db()
if db.has_rows(table):
    db.drop_triggers(table)
    ...
```
We can make that change later if we care to.